src/command_modules/azure-cli-cosmosdb/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+0.2.11
+++++++
+* Added support for setting "isZoneRedundant" property of location. Introduced new format for specifying location details of an account and deprecated the old format.
```

Comment:
Better to be more specific here. `* cosmsodb create/update: Added new format for --location to allow setting "isZoneRedundant" property. Deprecated old format.`